examples/voice_send.cr

```diff
@@ -50,7 +50,7 @@ client.on_message_create do |payload|
     # The DCAParser class handles parsing of the DCA file. It doesn't do any
     # sending of audio data to Discord itself – that has to be done by
     # VoiceClient.
-    parser = Discord::DCAParser.new(file)
+    parser = Discord::DCAParser.new(file, strict_metadata: false)
```

Comment:
This example isn't meant to make some statement about the point of a spec, it's meant to be usable in a real world situation where people might not care about the metadata and leave it empty.